packages/commonwealth/server/migrations/20231030172206-cosmos-hex-string.js

```diff
@@ -0,0 +1,84 @@
+'use strict';
+
+module.exports = {
+  up: async (queryInterface) => {
+    const { toHex, fromBech32 } = await import('@cosmjs/encoding');
+    const getHex = async (address) => {
+      const encodedData = fromBech32(address).data;
+      return toHex(encodedData);
+    };
+
+    await queryInterface.sequelize.transaction(async (t) => {
+      let bulkUpdateData = [];
+
+      await queryInterface.sequelize.query(
+        `
+        ALTER TABLE "Addresses" ADD COLUMN IF NOT EXISTS "hex" VARCHAR(64) NULL;
+        `,
+        { raw: true, transaction: t }
+      );
+
+      // get all cosmos addresses and assign a hex string
+      const [addresses] = await queryInterface.sequelize.query(
+        `
+        SELECT *
+        FROM "Addresses"
+        WHERE "wallet_id" IN ('keplr', 'cosm-metamask', 'terrastation', 'keplr-ethereum');     
+          `,
+        { transaction: t }
+      );
+
+      for (const address of addresses) {
```

Comment:
This takes the remaining half of the execution time. So this is another good candidate to optimize